StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheet+DeferredAPI.swift

```diff
@@ -48,6 +48,13 @@ extension PaymentSheet {
                 switch intentConfig.mode {
                 case .payment:
                     let paymentIntent = try await configuration.apiClient.retrievePaymentIntent(clientSecret: clientSecret, expand: ["payment_method"])
+                    let completion = { (status: STPPaymentHandlerActionStatus, paymentIntent: STPPaymentIntent?, error: NSError?, deferredIntentConfirmationType: STPAnalyticsClient.DeferredIntentConfirmationType) in
```

Comment:
i think we should rename this `completion` to avoid masking the existing `completion` pass in as an argument to the function.

Reply:
I think it be clearer to avoid overriding the completion variable and calling it wrappedCompletion, but I'm not going to block on this because the two completion handlers have different arguments.